### ISSUE TITLE
[Завдання] The message of success appears when you try to save a new task without choosing a challenge

### ISSUE THREAD

**Issue by VitaliiPopovych1408** [ita-social-projects/TeachUA#1337]
**Environment:** Windows 10 Professional, Chrome 96.0.4664.110
**Reproducible:** always
**Build found:**  https://speak-ukrainian.org.ua/dev/

**Preconditions**
1. Log in as an administrator on https://speak-ukrainian.org.ua/dev/ (login: [admin@gmail.com](mailto:admin@gmail.com); password: admin)

**Steps to reproduce**
1. Go to user cabinet at top right corner - "Адміністрування" - "Завдання"
2. Click on the "Додати завдання" button
3. Press "Зберегти" button
4. Pay attention to the success message at the top part of the webpage

**Actual result**
The message about the success of the creation of a new task appears after clicking the "Save" button and with the blank field "Челендж".

**Expected result**
Admin can create new task only in case, when all fields filled with valid data.

![1](https://user-images.githubusercontent.com/42620939/155084934-cc0c5e27-bfae-42f0-b758-786a1e84b754.png)
![2](https://user-images.githubusercontent.com/42620939/155084939-8504457b-fc7d-4d1b-98ba-fb428b2b59fe.png)
![завдання без челенджу](https://user-images.githubusercontent.com/42620939/155085002-65e31679-d2b1-485e-aa0e-8fc3162a3068.png)

User story#1335

**Comment by dtyta** [comment]
Tested. "Додати Завдання" possible only after filling in the fields with valid values.
